components/connector/internal/gqlschema/schema.graphql

```diff
@@ -0,0 +1,50 @@
+# Token
+type Token {
+    token: String! # eg.: "1edfc34g"
+}
+
+# Certificates
+type Certificates {
+    certificate: String!
+    caCertificate: String!
+    clientCertificate: String!
+}
+
+# ManagementPlaneInfo
+type ManagementPlaneInfo {
+    directorURL: String! # eg.: "https://director.cluster.kyma.cx/graphql"
+}
+
+# CSRInfo
+type CertificateSigningRequestInfo {
+    managementPlaneInfo: ManagementPlaneInfo!
+    subject: String! # eg.: "OU=Test,O=Test,L=Blacksburg,ST=Virginia,C=US,CN={ID}"
+    keyAlgorithm: String! # eg.: rsa2048
+}
+
+type Query {
+    # Client-Certificates
+    
+    # Retuns subject that should be placed in the signing request
```

Comment:
Returns

Reply:
Please write comments in `"""comment"""` form, to include it in generated API documentation